### PR TITLE
docs: broken links for edit page modified

### LINKED ISSUE #38: 🐛 Bug: "Edit this page" links are broken in projects
### Bug Report Checklist

- [X] I have [searched for related issues](https://github.com/LearningTypeScript/projects/issues?q=is%3Aissue+label%3A%22type%3A+bug%22) and found none that matched my issue.
- [X] This is the appropriate issue form for the bug I would like to report.

### Expected

In pages like https://www.learningtypescript.com/from-javascript-to-typescript, _Edit this page_ takes you to URLs like https://github.com/LearningTypeScript/projects/tree/main/src/content/external/projects/from-javascript-to-typescript/README.md. That's a 404.

### Actual

They should go cross-repo to URLs like https://github.com/LearningTypeScript/projects/blob/main/projects/from-javascript-to-typescript/README.md.

### Additional Info

_No response_

### PR DESCRIPTION
**Pull request by ssavi-ict**
## PR Checklist

- [x] Addresses an existing open issue: fixes #38 
- [x] That issue was marked as [accepting prs](https://github.com/LearningTypeScript/site/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
- [ ] Steps in [CONTRIBUTING.md](https://github.com/LearningTypeScript/site/blob/main/.github/CONTRIBUTING.md) were taken

## Overview
- Broken URL modified.

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ const config = {
         docs: {
           editUrl: "https://github.com/LearningTypeScript/projects/tree/main/",
           include: ["*/*.md", "*/*/*.md"],
-          path: "src/content/external/projects",
+          path: "projects",
```
Comment (JoshuaKGoldberg) [inline docusaurus.config.js:46]: Hmm, I get this error when I try to `yarn start` locally:

```
[INFO] Starting the development server...
[ERROR] Error: The docs folder does not exist for version "current". A docs folder is expected to be found at projects.
    at getVersionMetadataPaths (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/files.js:125:15)
    at async createVersionMetadata (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:94:68)
    at async Promise.all (index 0)
    at async readVersionsMetadata (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/versions/index.js:146:30)
    at async Object.pluginContentDocs [as plugin] (/home/josh/repos/site/node_modules/@docusaurus/plugin-content-docs/lib/index.js:32:30)
    at async initializePlugin (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/init.js:83:32)
    at async Promise.all (index 0)
    at async initPlugins (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/init.js:91:21)
    at async loadPlugins (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/plugins/index.js:26:21)
    at async load (/home/josh/repos/site/node_modules/@docusaurus/core/lib/server/index.js:77:58)
    at async start (/home/josh/repos/site/node_modules/@docusaurus/core/lib/commands/start.js:41:19)
error Command failed with exit code 1.
```

Are you able to run this PR?

**Review by JoshuaKGoldberg — CHANGES_REQUESTED** [review]
Looks like this is broken? 🤔

**Comment by JoshuaKGoldberg** [comment]
Thanks for sending this in!

Just a heads up: it's typical to check off the `[ ]` checkboxes in the description like `[x]` to indicate they're done. I edited it and did the first one. 🙂

File: docusaurus.config.js (hunk 1 above)
Comment (ssavi-ict) [reply docusaurus.config.js:46]: @JoshuaKGoldberg I tried to run using `yarn start` but was having the same error. 
I am new to working with `docusaurus`. 
Also, I checked the documentation of `docusaurus`.  In the docs versioning, they mentioned having a `docs/` directory. But I can not find any directory named `docs` in this repository. 

Can you enlighten how to fix this up. I tried to google with the error, could not find much resource.

Comment (JoshuaKGoldberg) [reply docusaurus.config.js:46]: > checked the documentation ... mentioned having a `docs/` directory

Could I suggest doing a deeper read of the docs, and trying out making a Docusaurus site or two of your own to try it out? This project configures docs using a pretty standard `docusaurus.config.js` line: https://github.com/LearningTypeScript/site/blob/2f8e8863d5fde61500c5c642b84ef5312f13e583/docusaurus.config.js#L43

This issue around configuring Docusaurus might not be a _good first issue_ for you if you're not familiar with Docusaurus. 🙂

**Comment by JoshuaKGoldberg** [comment]
https://github.com/LearningTypeScript/site/pull/47#issuecomment-1149229845, sorry!

**Comment by ssavi-ict** [comment]
@JoshuaKGoldberg It is completely OK. I had learnings. I wish to implement those learnings in the coming days. I appreciate your help and guidelines for the beginners like me. It will surely help me. 
Thank You.

**Pull request closed without merging**